src/pages/WorkspaceDetails.tsx

```diff
@@ -1,3 +1,116 @@
+import Sidebar from '../components/shared/Sidebar'
+import WorkSpaceSidebar from '../components/ui/WorkSpaceSidebar'
+import { Download, ChevronDown, Plus, Pen } from 'lucide-react'
+import UsersTable from '../components/ui/UsersTable'
+import { columns } from '../components/ui/columns'
+import { TableUser } from '@/util/interfaces'
+import {
+    Collapsible,
+    CollapsibleContent,
+    CollapsibleTrigger,
+} from '@/components/shadcn/collapsible'
+import { useLocation, Navigate } from 'react-router-dom'
 export default function WorkspaceDetails() {
-    return <div>WorkspaceDetails</div>
+    const { state } = useLocation()
+    if (!state) return <Navigate to="/workspace" />
+    const { name } = state
+    const data: TableUser[] = [
+        {
+            id: '728ed52f',
+            names: 'John Doe',
+            email: 'john.doe@example.com',
+        },
+        {
+            id: '728ed52f',
+            names: 'John Doe',
+            email: 'john.doe@example.com',
+        },
+    ]
+    const groups = [
+        {
+            id: '728ed52f',
+            name: 'Group 1',
+            projects: [
+                {
+                    id: '728ed52f',
+                    names: 'Project 1',
+                },
+                {
+                    id: '728ed52f',
+                    names: 'Project 2',
+                },
+            ],
```

Comment:
why do we have static data here ? I would suggest to create json files and put the static data there just so to make it easier to remove them in the future